docs/install_guides/pypi-openvino-dev.md

```diff
@@ -97,6 +93,19 @@ pot -h
 
 You will see the help message for Post-Training Optimization Tool if installation finished successfully.
 
+## Troubleshooting
+
+#### Error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio"
+
+On Windows* some dependencies may require compilation from source when installing. To resolve this issue you are required to install [Build Tools for Visual Studio* 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and repete package installation.
```

Comment:
To resolve this issue you are required to install ... and repete package installation ---> To resolve this issue, you need to install .... and repeat package installation